CMakeLists.txt

```diff
@@ -137,14 +137,23 @@ elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel")
   set(EXTRA_Fortran_FLAGS "")
 endif()
 
+# Logic for showing and handling warnings
 if(ENABLE_ALL_WARNINGS)
+  # GCC, Clang, and Intel seem to accept these
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic")
     if (NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel")
+      # ifort doesn't like -Wall
       set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -Wall")
     else()
+      # ifort always reports some diagnostics we don't necessarily care about
```

Comment:
Minor cleanup, the line is changing `CXX` compiler flags and not `ifort`.

Reply:
Good catch.